src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -49,8 +50,10 @@ public class StrykerOptions
 
         private const string ErrorMessage = "The value for one of your settings is not correct. Try correcting or removing them.";
         private readonly IFileSystem _fileSystem;
+        private readonly ILogger _bufferLogger;
```

Comment:
Why is the logger called buffelLogger? We just named it logger in all other classes.